src/WindowsAzureMessaging/WindowsAzureMessaging/Helpers/MSLocalStorage.m

```diff
@@ -0,0 +1,66 @@
+//----------------------------------------------------------------
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//----------------------------------------------------------------
+
+#import "MSLocalStorage.h"
+#import <Foundation/Foundation.h>
+
+static NSString *const installationKey = @"installation";
+static NSString *const lastInstallationKey = @"lastInstallation";
+static NSString *const enabledKey = @"notificationHubEnabled";
+
+@implementation MSLocalStorage
+
++ (BOOL)isEnabled {
+    NSUserDefaults *defaults = [NSUserDefaults standardUserDefaults];
+    NSData *encodedObject = [defaults objectForKey:enabledKey];
```

Comment:
Why not use `[defaults boolForKey:enabledKey]` directly? Serializing the bool value as an object seems like overkill.

Reply:
If the key isn't present, it returns NO, and instead following the AppCenter approach storing as a number.